tutorials/tmva/TMVA_CNN_Classification.C

```diff
@@ -107,14 +107,14 @@ void MakeImagesTree(int n, int nh, int nw)
    f.Close();
 }
 
-void TMVA_CNN_Classification(std::vector<bool> opt = {1, 1, 1, 1})
+void TMVA_CNN_Classification(std::vector<bool> opt = {0, 0, 0, 0, 1})
```

Comment:
I would add here {1,1,1,1,0}   
switch off pytorch by default since nobody has it installed

Reply:
And add also off at line 147